MinatureNoteBookPortalApi/Controllers/Miniature.cs

```diff
@@ -1,18 +1,18 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using MinatureNoteBookPortalApi.Data;
-using MinatureNoteBookPortalApi.Models;
+using MinatureNoteBookPortalApi.Dtos.Miniature;
 using MinatureNoteBookPortalApi.Mappers;
     using System.Xml;
 
 namespace MinatureNoteBookPortalApi.Controllers
 {
-    [Route("api/[miniature]")]
+    [Route("api/[controller]")]
     [ApiController]
-    public class Miniature : ControllerBase
+    public class MiniatureController : ControllerBase
     {
         private readonly AplicationDBContext _context;
-        public Miniature(AplicationDBContext context)
+        public MiniatureController(AplicationDBContext context)
         {
             _context = context;
```

Comment:
We try to avoid infrastructure in controllers. Its better to create extra layer here and access data via service.